### PR TITLE
UpdateDoc MicrosoftDocs/azure-docs-powershell#1266

### PR DESCRIPTION
**Pull request by MonikaReddy-MSFT**
Updating the Managed Identity Section of the doc MicrosoftDocs/azure-docs-powershell#1266

### CODE REVIEW

File: docs-conceptual/azps-3.1.0/authenticate-azureps.md [hunk 1]
```diff
@@ -110,8 +110,11 @@ $store.Close()
 Managed identities are a feature of Azure Active Directory. Managed identities are service principals assigned to resources that run in Azure. You can use a managed identity
 service principal for sign-in, and acquire an app-only access token to access other resources. Managed identities are only available on resources running in an Azure cloud.
 
-To learn more about managed identities for Azure resources, see
-[How to use managed identities for Azure resources on an Azure VM to acquire an access token](/azure/active-directory/managed-identities-azure-resources/how-to-use-vm-token).
+This command connects using the managed service identity of the host environment (for example, if executed on a VirtualMachine with an assigned Managed Service Identity, this will allow the code to login using that assigned identity)
```
Comment (aelij) [inline docs-conceptual/azps-3.1.0/authenticate-azureps.md]: It's been renamed from MSI to Managed Identity

**Comment by MonikaReddy-MSFT** [comment]
#sign-off

File: docs-conceptual/azps-3.1.0/authenticate-azureps.md (hunk 1 above)
Comment (MonikaReddy-MSFT) [reply docs-conceptual/azps-3.1.0/authenticate-azureps.md]: Renamed from MSI to Managed Identity

**Comment by MonikaReddy-MSFT** [comment]
#sign-off

**Comment by aelij** [comment]
#sign-off

**Comment by MonikaReddy-MSFT** [comment]
Since the changes are not merged yet, re-opening the pull request

**Comment by sdwheeler** [comment]
#do-not-merge

**Pull request closed without merging**